files/jira.rb

```diff
@@ -60,9 +60,11 @@ def close_issue(output, project)
         url = get_options[:site] + '/browse/' + issue.key
         puts "Closing Issue: #{issue.key} (#{url})"
 
+        occurrences = @event['occurrences'] rescue 'unknown'
+
         # Let the world know why we are closing this issue.
         comment = issue.comments.build
-        comment.save(:body => "This is fine:\n{code}#{uncolorize(output)}{code}")
+        comment.save(:body => "This is fine (occurrences=#{occurrences}):\n{code}#{uncolorize(output)}{code}")
```

Comment:
Can you start a trend by putting this extra data *after* the output? Mostly to give the most important info first and the extra info later.

We'll start with occurrences but there is plenty of room for lots more debug data.